src/main/java/net/snowflake/client/core/arrow/BigIntToFixedConverter.java

```diff
@@ -7,13 +7,15 @@
 import java.nio.ByteBuffer;
 import net.snowflake.client.core.DataConversionContext;
 import net.snowflake.client.core.SFException;
+import net.snowflake.client.core.SnowflakeJdbcInternalApi;
 import net.snowflake.client.jdbc.ErrorCode;
 import net.snowflake.client.jdbc.SnowflakeType;
 import net.snowflake.client.jdbc.SnowflakeUtil;
 import org.apache.arrow.vector.BigIntVector;
 import org.apache.arrow.vector.ValueVector;
 
 /** Data vector whose snowflake logical type is fixed while represented as a long value vector */
+@SnowflakeJdbcInternalApi
```

Comment:
I'm not sure we can change visibility of existed public members ....

Reply:
I'm not sure I agree - this annotation does not change visibility - it only marks API as internal. Code still compiles, right?

Reply:
You're kinda both right guys, reverted this change.